core/src/main/scala/chisel3/Aggregate.scala

```diff
@@ -258,25 +269,54 @@ sealed class Vec[T <: Data] private[chisel3] (gen: => T, val length: Int) extend
   private[chisel3] final override def allElements: Seq[Element] =
     (sample_element +: self).flatMap(_.allElements)
 
-  /** Strong bulk connect, assigning elements in this Vec from elements in a Seq.
+  /** The "bulk connect operator", assigning elements in this Vec from elements in a Seq.
     *
-    * @note the length of this Vec must match the length of the input Seq
+    * For chisel3._, uses the [[BiConnect]] algorithm; sub-elements of `that` may end up driving sub-elements of `this`
+    *  - Complicated semantics, will likely be deprecated in the future
```

Comment:
```suggestion
    * 
    *  - Complicated semantics, will likely be deprecated in the future
```

scaladoc nit to have the high-level description with a newline after it

Reply:
I'm confused, this is a subcommand after the big first line description?

Reply:
oh yeah my mistake